src/pages/Explore/explore.css

```diff
@@ -1,77 +1,171 @@
-.explore-main .container, .watch-later-main .container {
-    gap: var(--space-3);
-    padding: var(--space-3) var(--space-1)
+.explore-main .container,
+.watch-later-main .container,
+.playlists-main .container,
+.playlists-videos-list-main .container {
+	gap: var(--space-3);
+	padding: var(--space-3) var(--space-1);
 }
 
 .category-list {
-   gap: var(--space-1);
+	gap: var(--space-1);
 }
 
 .category-list .badge {
-    position: relative;
+	position: relative;
 }
 
 .videos-list-container {
-    display: grid;
-    width: 100%;
-    grid-template-columns: 1fr;
-    gap: var(--space-3) var(--space-2);
+	display: grid;
+	width: 100%;
+	grid-template-columns: 1fr;
+	gap: var(--space-3) var(--space-2);
 }
 
 .App.dark .video-card:hover {
-    background-color: var(--dark-theme-navbar-bg-color);
+	background-color: var(--dark-theme-navbar-bg-color);
 }
 
 .App.light .video-card:hover {
-    background-color: var(--light-theme-navbar-bg-color);
+	background-color: var(--light-theme-navbar-bg-color);
 }
 
 .video-card {
-    border-radius: var(--br-2);
-    -webkit-border-radius: var(--br-2);
-    -moz-border-radius: var(--br-2);
-    -ms-border-radius: var(--br-2);
-    -o-border-radius: var(--br-2);
-    background-color: inherit;
-    width: 100%;
-    height: 100%;
-    max-width: 100%;
-    transition: all 0.3s ease-in-out;
-    -webkit-transition: all 0.3s ease-in-out;
-    -moz-transition: all 0.3s ease-in-out;
-    -ms-transition: all 0.3s ease-in-out;
-    -o-transition: all 0.3s ease-in-out;
+	border-radius: var(--br-2);
+	-webkit-border-radius: var(--br-2);
+	-moz-border-radius: var(--br-2);
+	-ms-border-radius: var(--br-2);
+	-o-border-radius: var(--br-2);
```

Comment:
is it necessary to define border radius for different browsers

Reply:
Yes, I understand that browsers nowadays take care of this now. 
I use an extension that automatically does this, and on removing it, the lines are added again.